src/get-started/install/_get-sdk-linux.md

```diff
@@ -2,114 +2,99 @@
 
 {% include docs/china-notice.md %}
 
-On Linux, you have two ways you can install Flutter.
+To install the Flutter SDK on your Linux system, use one of the following methods.
 
-### Install Flutter using snapd
+### Method 1: Install Flutter using snapd
 
-The easiest way to install Flutter on Linux is by
```

Comment:
Why does a PR to change colors contain changes to content?

Reply:
Accidental inclusion of another PR. I'll fix it